main/reference/zoe-api/zoe.md

```diff
@@ -157,6 +160,22 @@ const bundle = await bundleSource(pathResolve(`./src/contract.js`));
 const installationP = await E(Zoe).install(bundle);
 ```
 
+## E(Zoe).installBundleID(bundleId, bundleLabel?)
+
+- **bundleId**: **BundleId**
+- **bundleLabel**: string - Optional
+- Returns: **Promise&lt;Installation>**
+
+Takes a bundleId for a Zoe contract (often generated in a [coreEval
+proposal](/guides/coreeval/local-testnet.html#deploying-contracts-using-core-eval-proposals)
```

Comment:
coreEval happens on chain. Bundle generation happens off-chain in a builder script.

I suggest linking to [bundling a contract](https://docs.agoric.com/guides/zoe/contract-walkthru.html#bundling-a-contract) instead.

Reply:
done